Add support for partition cloning (#2131)
OSInside/kiwi: doc/source/image_description/elements.rst

  Additional kernel parameters passed to the kernel by the
  bootloader.

root_clone="number"

>   For oem disk images, specifies to create `number` clone(s) of

Reviewer: ```suggestion
  For oem disk images, this element allows to create `number` clone(s) of
```

  the root partition, with `number` >= 1. A clone partition is

>   content wise an exact byte copy of the origin root partition.

Reviewer: ```suggestion
  content wise an exact byte for byte copy of the origin root partition.
```

  However, to avoid conflicts at boot time the UUID of any

>   cloned partition will be unique. In the sequence of partitions,

Reviewer: ```suggestion
  cloned partition will be made unique. In the sequence of partitions,
```
